### PR TITLE
Return first employee if multiple non 'prime role' employee records are returned

### PR DESCRIPTION
**Pull request by Chris-Thornton-Harvard**

### CODE REVIEW

File: qualtrics_link/util.py [hunk 1]
```diff
@@ -226,10 +226,13 @@ def filter_person_list(person_list):
             employee_list.append(person)
 
     # If more than one Person with Employee role, check if any have prime indicator set to 'Y'
+    # Else return the first employee record
     if len(employee_list) > 0:
```
Comment (sapnamysore) [inline qualtrics_link/util.py]: This handles multiple employee records, but not when there is only one employee record being returned.

Essentially, if len(employee_list)  is  0, return employee_list[0] else [contininue on with logic in line 230]

Comment (Chris-Thornton-Harvard) [inline qualtrics_link/util.py]: If there is one record returned, we check if the prime role is set to 'Y' if not we are still returning the first item in that list.
len(list('a')) will return 1 not 0

Comment (sapnamysore) [inline qualtrics_link/util.py]: Well, if rec1 is employee with prime_role not set, and rec2 is student with prime_role set, we will get res 2 with teh logic above. we want to return  rec1.

Comment (sapnamysore) [inline qualtrics_link/util.py]: Walked thru all test cases. all set. my bad, I was reading line 230 to be >1 not >0.

**Pull request merged**